tfs/src/main/java/hudson/plugins/tfs/TeamPushCause.java

```diff
@@ -12,19 +12,22 @@
 public class TeamPushCause extends SCMTriggerCause {
 
     private final String pushedBy;
+    private final String runConfig;

```

Comment:
Can you refactor this name to "context" just for consistency with other plugins that do similar things?

Reply:
Sure, refactored it "context".